File: updraft_sdk/src/main/java/com/apsswithlove/updraft_sdk/api/response/CheckLastVersionResponse.java
package com.apsswithlove.updraft_sdk.api.response;

import com.google.gson.annotations.SerializedName;

/**
 * Created by satori on 3/27/18.
 */

public class CheckLastVersionResponse {

    @SerializedName("create_at")
    private String mCreteAt;

    @SerializedName("whats_new")
    private String mWhatsNew;

    @SerializedName("version")
    private String mVersion;

    @SerializedName("your_version")
    private String mYourVersion;

    @SerializedName("is_new_version")
    private boolean mIsNewVersion;

    @SerializedName("is_autoupdate_enabled")
    private boolean mIsAutoupdateEnabled;

    public String getCreteAt() {
        return mCreteAt;
    }

    public String getWhatsNew() {
        return mWhatsNew;
    }

    public String getVersion() {
        return mVersion;
    }

    public String getYourVersion() {
        return mYourVersion;
    }

    public boolean getIsNewVersion() {
        return mIsNewVersion;
    }

    public boolean isAutoupdateEnabled() {
        return mIsAutoupdateEnabled;
    }
}

File: updraft-sdk/src/main/java/com/appswithlove/updraft/api/response/CheckLastVersionResponse.kt
package com.appswithlove.updraft.api.response

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class CheckLastVersionResponse(
    @SerialName("create_at") val createAt: String? = null,
    @SerialName("whats_new") val whatsNew: String? = null,
    @SerialName("version") val version: String? = null,
    @SerialName("your_version") val yourVersion: String? = null,
    @SerialName("is_new_version") val isNewVersion: Boolean,
    @SerialName("is_autoupdate_enabled") val isAutoupdateEnabled: Boolean,
)
